### ISSUE TITLE
[Held requests] break held message into separate system message + comment

### ISSUE THREAD

**Issue by robertjchen** [Expensify/App#36979]
## Action Performed:
break held message into separate system message + comment

## Expected Result:
![image](https://github.com/Expensify/App/assets/2438855/cfc563fb-fd0e-4db7-bd8b-a2e82173cb95)

## Actual Result:
<img width="488" alt="Screenshot 2024-02-20 at 5 34 03 PM" src="https://github.com/Expensify/App/assets/2438855/8aa6bf0d-08b7-4d06-a1eb-a0728bf11aba">

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by robertjchen** [comment]
@BartoszGrajdek when making the system message, it looks like we can simplify that further since we're breaking things up. I'll update the backend to split these into two different report actions, but the FE may need to generate the optimistic message 👍

**Comment by robertjchen** [comment]
working on updating the backend!

**Comment by robertjchen** [comment]
In progress, focused on getting partial pay/approve out first.

**Comment by robertjchen** [comment]
was blocked by https://github.com/Expensify/App/issues/36830

**Comment by robertjchen** [comment]
@BartoszGrajdek Could you please help me update the FE to do the following? Thanks!! 🙏 
- break the hold action into the hold action (without the comment) and a new optimistic comment.
- the `HoldRequest` API command will take a new `commentReportActionID` parameter for the optimistic comment report action.

**Comment by BartoszGrajdek** [comment]
Hi! Of course - I've created a draft PR with the changes mentioned above, not sure if the backend is fully working yet, so please let me know what's the status there 🙏🏻

**Comment by robertjchen** [comment]
Worked great! Some final backend tweaks were required to make things tick- the changes are now under review and will hopefully be merged and deployed to staging later today ([PR](https://github.com/Expensify/Web-Expensify/pull/41392)). Assuming the backend Web change goes out to staging today I'll go ahead and merge your branch first thing tomorrow 🙏

**Comment by robertjchen** [comment]
All PRs merged!! 🎉
